recipes/gstlal-inspiral/meta.yaml

```
@@ -0,0 +1,98 @@
+{% set name = "gstlal-inspiral" %}
+{% set version = "1.6.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz"
+  sha256: 66ef3f3f1552894cd6b1add4e49477e6a37c8adb23e5e1364fb5f001bfc57397
+  patches:
+    - missing-links.patch
+
+build:
+  ignore_run_exports:
+    # configure asks for it, but it is never actually used
+    - lalmetaio
+  number: 0
+  skip: true  # [win or py>=30]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make  # [unix]
+    - pkg-config >=0.18.0  # [unix]
+  host:
+    - glib
+    - gobject-introspection >=1.30.0
+    - gsl
+    - gstreamer
+    - gst-plugins-base
+    - gstlal >=1.5.0
+    - lal >=6.19.0
+    - lalinspiral >=1.8.0
+    - lalmetaio >=1.4.0
+    - numpy 1.14
```

As on the other gstlal PR:
```suggestion
    - numpy
```

---

@chrisburr, I had to add the `1.14` just for staged-recipes, otherwise the build tries to use `numpy 1.9` and I get a pile of conflicts, see [this build failure](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=119988&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0) as an example.

---

I plan to remove it again as soon as this gets as far as a feedstock.

---

I believe this is to blame:

https://github.com/conda-forge/staged-recipes/blob/4f830ffc8553912789e723a942b3b693c703ccb9/.circleci/build_steps.sh#L40

---

Of course...the staged-recipes CI needs some love at some point as there are now a lot of these quirks...